.github/workflows/fake_status.yml

```diff
@@ -0,0 +1,38 @@
+name: Simulate Success for Release PR
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
```

Comment:
Why is it run two times, i.e. release_chore and previous_version:

Reply:
The one is when a PR is first opened, the other is when the PR head is updated.

Reply:
Don't you need some `if` condition on the second ("previous") one?

Reply:
Oh yes. Because the simulation of Lint Test / lint_test should only occur if the release chore closes. For closing other PRs we don't need it although I made a demo to mark lint_test (i.e. the old status check to allow new PRs to pass)

Reply:
To go back to the two tasks: The one simulates "Lint Test / lint_test" which is the new name of the build status and needs to be added to the release chore PR. The other one simulates "lint_test" which is the current name of the build status which would cause this PR to fail until we switch to the other status.